lib/predictions/repo.ex

```diff
@@ -71,7 +71,12 @@ defmodule Predictions.Repo do
     end
   end
 
-  @decorate cacheable(cache: @cache, on_error: :nothing, opts: [ttl: @ttl])
+  @decorate cacheable(
+              cache: @cache,
+              on_error: :nothing,
+              match: fn lst -> lst != [] end,
```

Comment:
Since it overrides the default matcher, wouldn't this mean this starts caching error tuples and such too?

Reply:
`fetch` only returns lists but, I can add a guard in there.